### PR TITLE
Adding support for enterprisey GitHub

### PR DESCRIPTION
**Pull request by dbgrandi**
Needs some passing specs before it's a wizard.

6d2bf14c411

<boop>

### CODE REVIEW

**Comment by dbgrandi** [comment]
dcd1f340844

File: lib/danger/commands/local.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'octokit'
```
Comment (orta) [inline lib/danger/commands/local.rb]: is this still required

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## Master
 
+* Nothing yet, erase me - orta
```
Comment (orta) [inline CHANGELOG.md]: breaking change

File: lib/danger/commands/local.rb (hunk 1 above)
Comment (dbgrandi) [inline lib/danger/commands/local.rb]: nope, that was mid refactor

**Comment by KrauseFx** [comment]
These are some great gifs :+1:

**Pull request merged**